code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -475,8 +475,8 @@ It is a modified Beretta 93R, and can fire three round burst or single fire. Whe
 //The first rule of monkey pistol is we don't talk about monkey pistol.
 
 /obj/item/weapon/gun/pistol/chimp
-	name = "\improper CHIMP70 pistol"
-	desc = "A powerful sidearm issued mainly to highly trained elite assassin necro-cyber-agents."
+	name = "\improper BANAN-007 pistol"
+	desc = "(CHIMP70) A sidearm an Armsdealer's Concord manufacturer was tasked with making. Who it was for? Good luck finding out."
```

Comment:
The older description was funnier.

Reply:
Lore is more important tho.

Reply:
It's an admin spawn gun that shoots chimps.
The new description could still be moved to the codex.

Reply:
Chimp is more epic I'd say.